raw-spans-grouper/src/main/resources/configs/common/application.conf

```diff
@@ -19,6 +19,10 @@ kafka.streams.config = {
   replication.factor = ${?REPLICATION_FACTOR}
   topic.cleanup.policy = "delete,compact"
 
+  producer.max.request.size = 10485760
+  default.production.exception.handler = org.hypertrace.core.kafkastreams.framework.exceptionhandlers.IgnoreProductionExceptionHandler
```

Comment:
If it only requires in production (both production properties) and not as a default, we should move this to helm override.